README.md

```diff
@@ -116,6 +116,33 @@ display.Show("Hello World!")
 display.VerifyWasCalledOnce().Show("Hello World!")
 ```
 
+Tracking the pegomock tool in your project
```

Comment:
Thinking about where to best insert this section, how about adding this section after https://github.com/petergtz/pegomock#installation? Would feel very natural to me to look for something like this here.

Reply:
done